wolox-books/src/app/screens/BooksList/components/book-item/book-item.component.ts

```diff
@@ -11,10 +12,16 @@ export class BookItemComponent implements OnInit {
   @Input() bookTitle: string;
   @Input() bookAuthor: string;
   readonly defaultImageSrc = 'assets/book-cover.png';
+  bookCounter: number;
 
-  constructor() { }
+  constructor(private shoppingCartService: ShoppingCartService) { }
 
   ngOnInit(): void {
+    this.shoppingCartService.bookCounter.subscribe(bookCount => this.bookCounter = bookCount);
```

Comment:
Are you sure yo need this here, I notice that you are only showing the counter in the navbar and you 
are already calling this service there.

Reply:
I need to subscribe to the observable in both components, because if I don't subscribe in the book-item, when I try to "add a new book" I wouldn't know to which books add a new one, so I have like "update" anytime the same books, that's why I need to subscribe in the navbar and also in the book-item components.